doc/COMPILING/COMPILING.md

```diff
@@ -632,7 +632,7 @@ There are reports of CDDA building fine on recent OpenBSD and FreeBSD machines (
 
 ### Building on FreeBSD/amd64 13.0 with the system compiler
 
-FreeBSD uses clang as the default compiler as of 10.0, and combines it with libc++ to provide C++14 support out of the box.
+FreeBSD uses clang as the default compiler as of 10.0, and combines it with libc++ to provide C++17 support out of the box.
```

Comment:
Are you sure that `libcxx` in FreeBSD 10.0 supports C++17?

Reply:
Whoops, a stupid mistake, I didn't read the man page for clang in FreeBSD 10.0, gotta go change it now